.github/workflows/pr_unittests.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
           key: r-${{ hashFiles('DESCRIPTION') }}
       - name: Install dependencies
         run: |
-          install.packages(c("remotes","devtools","readr","covidcast"))
+          install.packages(c("remotes","devtools","covidcast"))
```

Comment:
Is `covidcast` also something that should be listed in the "imports" section of the DESCRIPTION?

Reply:
Some thoughts:
- People could happily use this package without using any functionality that requires covidcast, so i'd rather not require installation of that package...?
- however, we should probably check for whether that package is installed before trying to use it, which I bet we don't right now.  i'd move to file a separate issue for that.

Seem reasonable?